packages/generator/src/generator.ts

```diff
@@ -226,7 +226,7 @@ export async function generateSourcesForService(
     sourceFile(
       serviceDir,
       `${entity.className}RequestBuilder`,
-      requestBuilderSourceFile(entity),
+      requestBuilderSourceFile(entity, service),
```

Comment:
if I see it right also for this and the `complexTypeSourceFile()` below only the version from the service is actually needed.